File: Assg1/Part2/Prog5.java
package Part2;

import java.util.Scanner;

public class Prog5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter character: ");
        char c = sc.next().charAt(0);
        c=Character.toLowerCase(c);

        if(c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u')
            System.out.println(c + " is a vowel.");
        else 
            System.out.println(c + " is a consonant");
    }
}

File: AssgJS/Part2/Prog5.js
//install: npm install prompt-sync
const prompt=require("prompt-sync")({sigint:true});

let char = prompt("Enter character: ");

function charOrVow(char){
    const vowels =['a','e','i','o','u'];
    
    if(vowels.includes(char.charAt(0).toLowerCase())) 
        return true;
    return false;
}

charOrVow(char) ? console.log('Its vowel') : console.log('Its consonant');